jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/LoadDockerStep.java

```diff
@@ -85,13 +86,21 @@ public Void call() throws ExecutionException, InterruptedException {
   private Void afterPushBaseImageLayerFuturesFuture()
       throws ExecutionException, InterruptedException, IOException {
     Image<CachedLayer> image = NonBlockingSteps.get(NonBlockingSteps.get(buildImageStep));
+    ImageReference targetImageReference =
+        buildConfiguration.getTargetImageConfiguration().getImage();
 
     // Load the image to docker daemon.
     buildConfiguration.getBuildLogger().lifecycle("Loading to Docker daemon...");
-    new DockerClient()
-        .load(
-            new ImageToTarballTranslator(image)
-                .toTarballBlob(buildConfiguration.getTargetImageConfiguration().getImage()));
+    DockerClient dockerClient = new DockerClient();
+    dockerClient.load(new ImageToTarballTranslator(image).toTarballBlob(targetImageReference));
+
+    // Tags the image with all the tags. This only needs to be done for additional tags, so this can
+    // be skipped if there is only one tag - the load above already loads that tag.
+    if (buildConfiguration.getAllTargetImageTags().size() > 1) {
+      for (String tag : buildConfiguration.getAllTargetImageTags()) {
```

Comment:
Not a big deal I guess, but this means we'll also retag with the tag that's already been docker loaded, correct?

Reply:
Yea, that's true. I'll just cut that out.